src/openrct2/ride/Vehicle.cpp

```diff
@@ -7481,21 +7484,12 @@ bool Vehicle::UpdateTrackMotionBackwards(const CarEntry* carEntry, const Ride& c
         {
             auto brakeSpeed = ChooseBrakeSpeed();
 
-            if (-(brakeSpeed << 16) > _vehicleVelocityF64E08)
+            if (-(brakeSpeed << kTrackSpeedShiftAmount) > _vehicleVelocityF64E08)
             {
                 acceleration = _vehicleVelocityF64E08 * -16;
             }
         }
 
-        if (trackType == TrackElemType::Booster)
```

Comment:
Is this block removed because `vehicleVelocityF64E08` is negative?

Reply:
yes